releases/v1.6.0.toml

```diff
@@ -59,4 +59,15 @@ support to increase overall compatibility and stability.
 * **Allow WithServices to use custom implementations** ([#5709](https://github.com/containerd/containerd/pull/5709))
 * **Support custom compressor for walking differ** ([#5735](https://github.com/containerd/containerd/pull/5735))
 
+### Release Updates
+
+#### containerd/CNI/runc/critools tar bundle
+
+The tar bundles released as `cri-containerd-*.tar.gz` contain a build of runc
+linked with a newer version of libseccomp. This version of runc will not work on
+on some distributions, such as RHEL 7 and Debian 10. These tar bundles are now
```

Comment:
I'm afraid that people just cut out "This version of runc will not work on
on some distributions, such as RHEL 7 and Debian 10" and spread disinformation 😅 

What about saying "This dynamically-linked build of runc will not work on
on some distributions, such as RHEL 7 and Debian 10. Users of such distributions are suggested to get a statically-linked runc binary from https://github.com/opencontainers/runc/releases"

Reply:
It would be also nice to say that `cri-containerd-*.tar.gz` is targeted for Ubuntu 18.04